dfastmi/gui/dialog_view.py

```diff
@@ -163,7 +184,9 @@ def _update_branch(self, data):
         self._case_description.setText(self._view_model.model.case_description)
 
         # Update branch and reach selection
+        self._branch.blockSignals(True)
```

Comment:
Maybe move this to a method, since it used multiple times in a similar way, also for reach.
e.g.:
```
self.set_combobox_label_without_signals(self._branch, data)
self.set_combobox_label_without_signals(self._reach, data)

def set_combobox_label_without_signals(self, combobox : QComboBox, text : str):
        combobox.blockSignals(True)
        combobox.setCurrentText(text)
        combobox.blockSignals(False)
```

Reply:
Corrected eventing in #99 and #101

Reply:
Discussed with Bert.